lib-injection/dedupe.py

```diff
@@ -1,57 +0,0 @@
-import collections
```

Comment:
nit: Should we just keep this script here, and re-add the step in the `prepare-oci-package.sh` file when we revisit? Or do you foresee the implementation changing drastically?